news/2022-05-12-osu-talk-event-overcoming-obstacles.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "osu! Talk Event: Overcoming Obstacles"
+date: 2022-05-19 08:00:00 +0000
+---
+
+
+Mark your calendars, set your alarms, and join us for the first ever osu! talk event (OTE), where you can make your voice heard.
+
+
+![](/wiki/shared/news/2022-05-12-osu-talk-event-overcoming-obstacles/ote-newspost-banner.png)
+
+
+Unlike many events that involve guest speakers, OTE is open for anyone to speak -- all you need is a story to share. Each OTE centers around a certain theme. For this OTE, all talks will center around the theme of **overcoming obstacles**.
+
+
+Here's a rundown of how OTE works:
+
+
+- **The event will last 3 days total,** from May 20 - 22, 2022.

```

Comment:
it's a very small style thing, but if the emphasis covers the entire sentence, then you include the punctuation

```suggestion
- **The event will last 3 days total**, from May 20 - 22, 2022.
```